tests/tape/tapes/test_tape.py

```diff
@@ -823,7 +823,7 @@ def test_single_expectation_value(self, tol):
 
     def test_multiple_expectation_values(self, tol):
         """Tests correct output shape and evaluation for a tape
-        with multiple expval outputs"""
+        with multiple qml.expval outputs"""
```

Comment:
```suggestion
        with multiple expval outputs"""
```
Now I'm going crazy and thinking that `qml.expval` reads fine here anyway :laughing:

Reply:
Ok from here I'm gonna stop

Reply:
That was a heroic effort, sorry for that Tom!